ingestion/src/metadata/cli/restore.py

```diff
@@ -0,0 +1,101 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Restore utility for the metadata CLI
+"""
+from pathlib import Path
+from typing import List, Optional
+
+import click
+from sqlalchemy import create_engine, inspect, text
+from sqlalchemy.engine import Engine
+
+from metadata.generated.schema.entity.services.connections.database.mysqlConnection import (
+    MysqlConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.postgresConnection import (
+    PostgresConnection,
+)
+from metadata.utils.connections import get_connection
+from metadata.utils.helpers import list_to_dict
+from metadata.utils.logger import cli_logger
+
+logger = cli_logger()
+
+
+def execute_query(engine: Engine, input: Path, schema: str = None) -> None:
```

Comment:
This is not really running a query right? It would be something like `execute_sql_file` 🤔